flocker/control/schema/types.yml

```diff
@@ -60,6 +60,32 @@ definitions:
     minimum: 1
     maximum: 1024
 
+  container_links:
+    title: "Container links"
+    description: |
+        A mapping of links between containers, specifying the local and remote ports.
+        See https://docs.clusterhq.com/en/latest/advanced/configuration.html for more information.
```

Comment:
As mentioned above, I think we should include actual docs here. Insofar as there is a link it should be to Docker link environment variable docs.